doc/rtd/reference/yaml_examples/fan.rst

```diff
@@ -0,0 +1,14 @@
+.. _cce-fan:
+
+Configure Ubuntu fan networking
```

Comment:
Should this be moved to the ubuntu specific section?

Reply:
Yeah I wasn't sure whether to put it in this section or the ubuntu one. I'll move it.